### DISCUSSION TITLE
how retrive the eval data type

### DISCUSSION THREAD

**Discussion by yongman** [mlua-rs/mlua#150] [Q&A] [answered]
How to retrieve the data type of chunk.eval？ because  I want to convert lua response data to other data protocol. 
Thanks.

**Comment by khvzak** [comment, marked as answer]
Do you have a code example? 
Any datatype is supported that implements [`FromLua`](https://docs.rs/mlua/0.7.4/mlua/trait.FromLua.html) trait

**Reply by yongman** [reply]
> Do you have a code example? 
> Any datatype is supported that implements [`FromLua`](https://docs.rs/mlua/0.7.4/mlua/trait.FromLua.html) trait

Thanks for your reply. My question has been resolved by using the Value enum type. Thanks again.
